electron/electron.js

```diff
@@ -91,10 +91,33 @@ ipcMain.handle('save-output', async (event, savePath, extractedData) => {
       const outputFile = path.join(savePath, `mcode-extraction-patient-${i + 1}.json`);
       fs.writeFileSync(outputFile, JSON.stringify(bundle), 'utf8');
     });
-    // retuerning true indicates that the save process succeeded
+    // returning true indicates that the save process succeeded
     return true;
   } catch (error) {
     // return the error message
     return error.message;
   }
 });
+
+ipcMain.handle('save-config-as', async (event, schema) => {
```

Comment:
This is pretty small, but I think it will help us understand the code in the future. What will be saved here isn't a JSON schema, but the JSON that someone built in the editor. I think rather than calling this variable `schema`, we should rename it to reflect that it is just JSON data, so maybe something like `config` or `configJSON`.

Reply:
I have renamed all of the schema variables to configJSON.